content/docs/command-reference/repro.md

```diff
@@ -105,11 +105,11 @@ up-to-date and only execute the final stage.
   `./dvc.yaml` by default). Different things can be provided as targets
   depending on the flags used (more details in each option). Examples:
 
-  - `dvc repro linear/dvc.yaml`: Specific `dvc.yaml` file to reproduce
-  - `dvc repro -R pipelines/`: Directory path to explore recursively for
-    `dvc.yaml` files
-  - `dvc repro train-model`: Specific stage in `./dvc.yaml`
-  - `dvc repro modeling/dvc.yaml:prepare`: Stage in a specific `dvc.yaml` file
+  - `dvc repro linear/dvc.yaml`: Specific `dvc.yaml` file
```

Comment:
let's not mix pls multiple things in one PR

Reply:
To my mind it's basically the same list in `repro`, so I edited both consistently. But sure, I will extract this ⌛

Reply:
Extracted to #2425